src/Serialization/Upgrades/1.4.0.jl

```diff
@@ -1,6 +1,7 @@
 push!(upgrade_scripts_set, UpgradeScript(
   v"1.4.0",
   function upgrade_1_4_0(s::UpgradeState, dict::Dict)
+    upgraded_dict = dict
```

Comment:
Side question, but: why do you do this? In all of the upgrade scripts I had a look at, `upgraded_dict` and `dict` are *identical*. There is no obvious reason why there are two names referencing to the same object. OK, on the *surface* lines like
```
          upgraded_dict[:data] = Dict(
            :symbols => dict[:data][:symbols],
          )
```
might *look* as if they are shuffling data from one dictionary into another, but that's actually actively misleading.